index.bs

```diff
@@ -384,6 +382,8 @@ When this method is invoked, the user agent MUST run the following steps:
               <dt> Otherwise
               <dd> Append |session| to the [=list of inline sessions=].
             </dl>
+        1. Let |session| be a new {{XRSession}} object.
+        1. [=Initialize the session=] with |session|, |mode|, and |device|.
```

Comment:
This should be moved one step up as you might potentially set the active immersive session to session (and or adding it to the list of inline sessions) just above